book/index.qmd

```diff
@@ -142,7 +142,7 @@ Finally, @sec-fairness will introduce algorithmic fairness, which includes speci
 
 As well as the editors and contributing authors, many others have contributed to this book.
 We would like to acknowledge Stefan Coors for creating many of the images in the book, as well as Daniel Saggau, Jakob Richter, and Marvin Böcker for contributions to earlier drafts the book.
-Many of the contributing authors were supported by the Munich Center for Machine Learning (MCML) and some by the National Science Foundation (NSF).
+Many of the contributing authors were supported by the Munich Center for Machine Learning (MCML) and some by the National Science Foundation (NSF) or the Mathematical Research Data Initiative (MaRDI).
```

Comment:
```suggestion
We would also like to acknowledge the following institutes that supported various contributors: Munich Center for Machine Learning (MCML), National Science Foundation (NSF), and Mathematical Research Data Initiative (MaRDI).
```

Reply:
I wouldn't really call MaRDI an institute, is NSF an institute?

Reply:
Would you prefer organisation?

Reply:
yes